automacent-fwk-components/automacent-fwk-core/src/main/java/com/automacent/fwk/core/TestObject.java

```diff
@@ -126,6 +126,30 @@ public void setDriverManager(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
 
+	// Debugger Address ----------------------------------------------------
+
+	private String debuggerAddress;
+
+	/**
+	 * set the debugger address
+	 * 
+	 * @param debuggerAddress
+	 */
+	public void setDebuggerAddress(String debuggerAddress) {
+		this.debuggerAddress = debuggerAddress;
+
+		_logger.info(String.format("debuggerAddress set to %s", getDebuggerAddress()));
+

```

Comment:
Add a check here to check if Browser is CHROME. If chrome then set, otherwise throw an warning log stating that the browser is not chrome and we will ignore debuggerAddress

Reply:
added the check and added an extra check as well to see if debuggerAddress is empty for chrome